docs/get-started-with-datahub.md

```diff
@@ -1,63 +1,21 @@
 # Get Started With DataHub
 
-This section is focused on helping you get up and running with DataHub as quickly as possible.
-
-## Get Started with Self-Hosted DataHub
-
-This is a great place to start if you're new to DataHub and are interested in taking the open-source project for a spin.
-
-You'll learn how to:
-
-* Deploy DataHub on your local machine
-* Ingest sample data to get a feel for what is possible in DataHub
-* Add and invite users to DataHub
-* Troubleshoot common issues
-
-### Prerequisites
-
-Before you go further, ensure you have the following installed:
-
-* [Python >=3.7.0](https://www.python.org/downloads/)
-* [Docker](https://docs.docker.com/get-docker/)
-* [jq](https://stedolan.github.io/jq/download/)
-* [Docker Compose](https://github.com/docker/compose/blob/master/INSTALL.md) - if using Linux
-
-:::note
-
-Make sure to allocate enough hardware resources for Docker! 
-We recommend the following configuration: 2 CPUs, 8GB RAM, 2GB Swap area, and 10GB disk space.
-
-:::
+These guides are focused on helping you get up and running with DataHub as quickly as possible.
 
 <p>
 <a
+    align="left"
     className='button button--primary button--lg'
+    style={{ margin: 12 }}
     href="quickstart">
     Get Started with Self-Hosted DataHub
 </a>
-</p>
-
-## Get Started with Managed DataHub
-
-[Acryl Data](https://www.acryldata.io/product) provides a fully managed, premium version of DataHub.
```

Comment:
Same thing here, you may not even need to change this line, just cut out the "you'll learn how too" and the "prereqs" and go straight to the button